testdata/p4_16_samples/pna-dpdk-small_sample.p4

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
This example does not have a corresponding reference file. They need to be generated.